### PR TITLE
Add update pins pipelines

### PR DESCRIPTION
**Pull request by Brian-McM**
## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by bcreane — APPROVED** [review]

**Review by bcreane — APPROVED** [review]

**Pull request merged**
